app/views/dashboard/_header.html.erb

```diff
@@ -0,0 +1 @@
+<h1><%= @page_setup.header[:title] %></h1>
```

Comment:
since this is just one line - i'd probably just keep it in each individual file rather than setting a variable to handle this